components/brave_wallet_ui/constants/types.ts

```diff
@@ -201,6 +201,8 @@ export interface PageState {
   showIsRestoring: boolean
   importError: boolean
   showAddModal: boolean
+  isBraveCryptoWalletInstalled: boolean
```

Comment:
isCryptoWalletsExtnesionInstalled

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558